rpc/src/lib.rs

```diff
@@ -38,3 +38,24 @@ pub mod mayastor {
 
     include!(concat!(env!("OUT_DIR"), "/mayastor.rs"));
 }
+
+#[allow(dead_code)]
+#[allow(clippy::type_complexity)]
+#[allow(clippy::unit_arg)]
+#[allow(clippy::redundant_closure)]
+#[allow(clippy::upper_case_acronyms)]
+pub mod mayastorv1 {
```

Comment:
what if we tried something like this:
```
pub mod mayastor {
   mod v1 { ... }
   mod v2 { ..}
}
```

Then we can use it as mayastor::v1::Pool and mayastor::v2::Pool ?